### ISSUE TITLE
v1.2.2 fails while trying to load a library

### ISSUE THREAD

**Issue by jeffbyrnes** [cprecioso/alfred-soulvercore#2]
After upgrading from v1.2.0 (works) to v1.2.2 (doesn’t work), I’m seeing these errors when trying to perform calculations:

```
[11:21:37.794] SoulverCore[Script Filter] Queuing argument '1'
[11:21:37.800] SoulverCore[Script Filter] Script with argv '1' finished
[11:21:37.801] ERROR: SoulverCore[Script Filter] Code 6: Exited with error code but no message
[11:21:40.498] SoulverCore[Script Filter] Queuing argument '1+'
[11:21:40.501] SoulverCore[Script Filter] Script with argv '1+' finished
[11:21:40.502] ERROR: SoulverCore[Script Filter] Code 6: Exited with error code but no message
[11:21:42.832] SoulverCore[Script Filter] Queuing argument '1+1'
[11:21:42.835] SoulverCore[Script Filter] Script with argv '1+1' finished
[11:21:42.836] ERROR: SoulverCore[Script Filter] Code 6: Exited with error code but no message
[11:21:46.525] SoulverCore[Script Filter] Queuing argument '1+'
[11:21:46.528] SoulverCore[Script Filter] Script with argv '1+' finished
[11:21:46.529] ERROR: SoulverCore[Script Filter] Code 6: dyld[12759]: Library not loaded: @rpath/SoulverCore.framework/Versions/A/SoulverCore
  Referenced from: /Users/jeffbyrnes/Dropbox/Library/Application Support/Alfred 4/Alfred.alfredpreferences/workflows/user.workflow.8E966B64-A58C-453F-983E-BB6154DF5BA4/alfred-soulvercore
  Reason: tried: '/Users/jeffbyrnes/Dropbox/Library/Application Support/Alfred 4/Alfred.alfredpreferences/workflows/user.workflow.8E966B64-A58C-453F-983E-BB6154DF5BA4/../lib/SoulverCore.framework/Versions/A/SoulverCore' (no such file), '/Users/jeffbyrnes/Dropbox/Library/Application Support/Alfred 4/Alfred.alfredpreferences/workflows/user.workflow.8E966B64-A58C-453F-983E-BB6154DF5BA4/../lib/SoulverCore.framework/Versions/A/SoulverCore' (no such file), '/Library/Frameworks/SoulverCore.framework/Versions/A/SoulverCore' (no such file), '/System/Library/Frameworks/SoulverCore.framework/Versions/A/SoulverCore' (no such file)
[11:21:46.702] SoulverCore[Script Filter] Queuing argument '1'
[11:21:46.706] SoulverCore[Script Filter] Script with argv '1' finished
[11:21:46.706] ERROR: SoulverCore[Script Filter] Code 6: dyld[12761]: Library not loaded: @rpath/SoulverCore.framework/Versions/A/SoulverCore
  Referenced from: /Users/jeffbyrnes/Dropbox/Library/Application Support/Alfred 4/Alfred.alfredpreferences/workflows/user.workflow.8E966B64-A58C-453F-983E-BB6154DF5BA4/alfred-soulvercore
  Reason: tried: '/Users/jeffbyrnes/Dropbox/Library/Application Support/Alfred 4/Alfred.alfredpreferences/workflows/user.workflow.8E966B64-A58C-453F-983E-BB6154DF5BA4/../lib/SoulverCore.framework/Versions/A/SoulverCore' (no such file), '/Users/jeffbyrnes/Dropbox/Library/Application Support/Alfred 4/Alfred.alfredpreferences/workflows/user.workflow.8E966B64-A58C-453F-983E-BB6154DF5BA4/../lib/SoulverCore.framework/Versions/A/SoulverCore' (no such file), '/Library/Frameworks/SoulverCore.framework/Versions/A/SoulverCore' (no such file), '/System/Library/Frameworks/SoulverCore.framework/Versions/A/SoulverCore' (no such file)
```

**Comment by cprecioso** [comment]
Hi! Two things to help me debug:
- Did you follow the 'Workaround' steps from https://github.com/cprecioso/alfred-soulvercore/releases/tag/v1.2.2 ?
- Are you on an Apple Silicon chip?

**Comment by cprecioso** [comment]
The new build system introduced on https://github.com/cprecioso/alfred-soulvercore/commit/b8798ab2affe4cb1cc35cc1c5809d0e96bc8b4df generates binaries with wrong `rpath` for searching the framework, on it

**Comment by cprecioso** [comment]
Fixed in commit 35fb4b97d548c2a6306e8b2e29435ccd7a9275fb, should work now if you use https://github.com/cprecioso/alfred-soulvercore/releases/tag/v1.2.3

**Comment by jeffbyrnes** [comment]
Thanks for the quick turnaround! Working great now.
